basilisk/src/grid/gpu/Benchmarks.md

```
@@ -0,0 +1,155 @@
+  workstation](https://apy-groupe.com).
+
+Do not hesitate to send [me](/sandbox/popinet/README) benchmarks
+results on other cards: to reproduce the benchmarks on your system
+follow the links for the raw scripts and results given in each
+section.
+
+## Time-reversed advection in a vortex
+
+This is this [test case](/src/test/advection.c) i.e. the [BCG](/src/bcg.h) advection solver.
+
+See [Benchmarks/advection]() for the commands and raw data.
```

markdownlint-cli2 (0.17.2)

No empty links
null

(MD042, no-empty-links)